cmd/loadTests.go

```diff
@@ -1050,13 +1109,17 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 							PipelineRunFailedTimeSumPerThread[threadIndex] += dur
 							logError(14, fmt.Sprintf("Pipeline run for applicationName/componentName %s/%s failed due to %v: %v", applicationName, componentName, succeededCondition.Reason, succeededCondition.Message))
 							FailedPipelineRunsPerThread[threadIndex] += 1
+							MetricsWrapper(MetricsController, metricsConstants.CollectorPipelines, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedPipelineRunsCreationCounter)
 						} else {
 							dur := pipelineRun.Status.CompletionTime.Sub(pipelineRun.CreationTimestamp.Time)
 							PipelineRunSucceededTimeSumPerThread[threadIndex] += dur
+							MetricsWrapper(MetricsController, metricsConstants.CollectorPipelines, metricsConstants.MetricTypeGuage, metricsConstants.MetricPipelineRunsTimeGauge, float64(dur))
```

Comment:
```suggestion
							MetricsWrapper(MetricsController, metricsConstants.CollectorPipelines, metricsConstants.MetricTypeGuage, metricsConstants.MetricPipelineRunsTimeGauge, dur.Seconds())
```
Instead of nanoseconds, please push seconds.